cdap-watchdog/src/test/java/co/cask/cdap/logging/framework/CDAPLogAppenderTest.java

```diff
@@ -154,6 +155,13 @@ public void testCDAPLogAppender() throws Exception {
       }
       logEventCloseableIterator.close();
       Assert.assertEquals(1, logCount);
+      // checking permission
+      String expectedPermissions = "rwx------";
```

Comment:
i am using "rwx" as execute permission is required in mac for create directory under a directory and it fails with access denied exception otherwise.